<!-- Launch the nodes and populate the ROS params needed for Spot to navigate -->
<launch>

    <!-- Indicate that the navigation system is active -->
    <param name="/spot/navigation/active" value="true" />

    <!-- YAML file specifying the 2D poses of named waypoints -->
    <param name="/spot/navigation/waypoints_yaml" value="$(find spot_skills)/config/waypoints.yaml" />

    <!-- Distance (meters) within which Spot is considered "close" to its goal -->
    <param name="/spot/navigation/close_to_goal_m" value="0.5" />

    <!-- Duration (seconds) after which a navigation command will time out -->
    <param name="/spot/navigation/timeout_s" value="120" />

    <!-- Absolute angle (radians) within which Spot is considered "close" to its goal (here, 90 degrees) -->
    <param name="/spot/navigation/close_to_goal_rad" value="1.5708" />

    <!-- Launch Spot's navigation stack using move_base -->
    <include file="$(find spot_move_base)/launch/move_base.launch"/>

</launch>